File: HFY.MvcWebUI.Angular/ClientApp/src/app/lazy-general-page/classes/firmalar/firmalar.ts
export class Firmalar {
  FirmaID: number;

  Rol: string;
  BagliOlduguID: number;
  FirmaAd: string;
  FirmaKisaAd: string;
  TelefonNumarasi: string;
  SMSTelefonNumarasi: string;
  GoogleMapsAdresi: string;
  VergiTCNo: string;
  FaturaAdresi: string;
  Adres: string;
  Eposta: string;
  PsiBar:string;
  DisDerinligiSayisi:number;
  ParaBirimID:number;
  KayitTarihi: Date;
  KullaniciGorevi:string;
  KullaniciKisaKod:string;
  YetkiliKisi:string;
  OlusturanId: number;
  OlusturmaTarihi: Date | string;
  DuzenleyenId: number;
  DuzenlemeTarihi: Date | string;
  Aktif: boolean;
  ListeAktiflik: boolean;

  jqueryDataTableFirmalar() {
    return [{ data: 'YetkiliKisi' }, { data: 'BagliOlduguID' }, { data: 'FirmaAd' }, //{ data: 'FirmaKisaAd' },
    { data: 'Aktif' } // işlemler
    ]
  }

  jqueryDataTableIsletmeler() {
    return [{ data: 'VergiTCNo' }, { data: 'FirmaAd' }, //{ data: 'FirmaKisaAd' },
    { data: 'Aktif' } // işlemler
    ]
  }

  // Kullanıcı listesini çekerken bunu çağırıyoruz.
  jqueryDataTableKullanicilar() {
    return [{ data: 'FirmaAd' }, { data: 'FirmaKisaAd' }, { data: 'KullaniciGorevi' },
    { data: 'Aktif' } // işlemler
    ]
  }
}

File: HFY.Entities/Concrete/Firmalar.cs
using HFY.Core.Entities;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace HFY.Entities.Concrete
{
    public class Firmalar: IEntity
    {
        [Key]
        public int FirmaID { get; set; }

        public string Rol { get; set; }
        public int BagliOlduguID { get; set; }
        public string FirmaAd { get; set; }
        public string FirmaKisaAd { get; set; }
        public string Sifre { get; set; }

        public string TelefonNumarasi { get; set; }
        public string SMSTelefonNumarasi { get; set; }
        public string GoogleMapsAdresi { get; set; }
        public string VergiTCNo { get; set; }
        public string FaturaAdresi { get; set; }
        public string Adres { get; set; }
        public string Eposta { get; set; }
        public string PsiBar { get; set; }
        public byte DisDerinligiSayisi { get; set; }
        public int ParaBirimID { get; set; }
        public DateTime? KayitTarihi { get; set; }

        public string KullaniciGorevi { get; set; }
        public string KullaniciKisaKod { get; set; }

        public string YetkiliKisi { get; set; }

        public int OlusturanId { get; set; }
        public DateTime OlusturmaTarihi { get; set; }
        public int DuzenleyenId { get; set; }
        public DateTime DuzenlemeTarihi { get; set; }
        public DateTime? SonGirisTarihi { get; set; }
        public bool Aktif { get; set; }
        public bool ListeAktiflik { get; set; }
    }
}

